litellm/proxy/proxy_server.py

```diff
@@ -2393,6 +2393,7 @@ async def completion(
             data["metadata"] = {}
         data["metadata"]["user_api_key"] = user_api_key_dict.api_key
         data["metadata"]["user_api_key_metadata"] = user_api_key_dict.metadata
+        data["metadata"]["user_api_key_alias"] = user_api_key_dict.key_alias
```

Comment:
can we use `getattr(user_api_key_dict, "key_alias", None)` to maintain backwards compatibility with versions that don't have key alias

Reply:
Yes, I'll do it